content/release/ReleaseNotes353.md

```diff
@@ -0,0 +1,39 @@
+<!--
+title: "Contrast 3.5.3 - June 2018"
+description: "Contrast 3.5.3 June 2018"
+tags: "3.5.3 June Release Notes"
+-->
+
+Congrats, Python Protect! You made it to the big leagues. 
+
+## Fixes
+
+* Assess rules take effect for child applications. 
+* Tags render correctly for IE11 users. 
+* Removed appearance of **Apply License** options for View-level users.
+* Use the link to clear Advanced filters in the Libraries page. 
+* Critical vulnerabilities disappear from your Dashboard as soon as you mark them "Not a Problem". 
+
+## Agent Updates
+
+### Java summary 
+
+Contrast now allows users to set the Application Code attribute of an application using configuration arguments passed to the Java agent. The Java team also added property `contrast.stderr` for directing Contrast logs to STDERR instead of a log file, improved accuracy for XSS detection, and fixed a deadlock at start-up which affects WebLogic 12. When bot-blocking is enabled, the Java agent will always block the Mozilla bot even when the user hasn't included any bots to block in their Contrast application configuration
```

Comment:
Can we say 

> by passing the `contrast.application.code` system property to the Java agent.

Instead of

> using configuration arguments passed to the Java agent